monash-human-power/dashboard
# Refactoring topics from topics.js to YAML topics file in common

## Description

Refactored the topics.js file to use the YAML topics file in the common repository. Also allowed display of the camera IP address.

## Screenshots

![image](https://user-images.githubusercontent.com/78238077/137614079-bf390888-8fbc-4aa0-9d24-eedd2133fb22.png)


## Steps to Test

1. Run the dasboard
2. Run `poetry shell` and then `python orchestrator.py` in the raspicam repository
3. Should display the camera system to be online, with the ip address
4. Other tests include running `mosquitto_pub -t "/v3/wireless_module/2/status" -h localhost -m "{\"online\": true}"` and then running `python -m das.V3_fake_module -t 10 -r 3` from the data-acquisition repository. Should be working the same without the topics file.
